demo/api/src/importer/pipes/end.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import { Transform, TransformCallback } from "stream";
+
+import { ImporterPipe } from "./importer-pipe.type";
+
+export class EndPipe implements ImporterPipe {
+    getPipe() {
+        return new EndHandler();
+    }
+}
+
+export class EndHandler extends Transform {
```

Comment:
I wouldn't export this (Knip will complain in `next`):

```suggestion
class EndHandler extends Transform {
```